slapp/app/index.liquid.html

```diff
@@ -210,11 +210,71 @@
     </classification-target>
     
     <full-instructions header="Cell Detection Instructions">
+        <h5>Objective</h5>
         <p>Decide whether the region of interest is a cell or not.</p>
-        <div>
-           <p><strong>Example: </strong></p>
-           <p><strong>Response: </strong></p>
-        </div>
+        <br>
+        <h5>Supported Browsers</h5>
+        <p>We currently only support modern Chrome and modern Firefox.
+          Support for other browsers will not be provided, use at your own
```

Comment:
I don't think things will even show up in Safari/Edge, so let's just remove the "use at your own risk" phrase.

Reply:
Removed the phrase!